aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include <ATen/ATen.h>
 #include <ATen/NestedTensorImpl.h>
+#include <ATen/native/NonSymbolicBC.h>
```

Comment:
I added them to the general `NativeFunctions.h` header (since they're part of `at::native::`) For the OSS builds though, that won't help because we use per-operator headers - so we still need to manually include them here.

That feels a bit weird, but should still accomplish the goal of reducing the risk of internal build failures.

Reply:
although - sigh - OSS mobile builds aren't recognizing `ATen/native/NonSymbolicBC.h`. I probably need to add it to a makefile somewhere.

Reply:
hmm @albanD - it looks like the build system is complaining because I'm trying to include a header from aten/native (`ATen/native/NonSymbolicBC.h`) inside of a header from plain ATen (`ATen/NativeFunctions.h`).

I think that logically, `NonSymbolicBC.h` and `NativeFunctions.h` should probably live in the same location, since they both refer to native variants of the aten API. That makes me think that `NonSymbolicBC.h` should be moved to live in `aten/src/ATen` instead of `aten/src/ATen/native`.

The whole point of the changes in this PR though were to reduce the risk of internal changes being needed, and that would definitely require an internal change, since that header is used internally now 😛.

So my current plan is just to ignore `NonSymbolicBC.h` entirely in this PR, since it's very unlikely that any of the `at::native::*_backward()` functions that I'm changing are actually directly called anywhere. LMK what you think!

Reply:
sgtm. keep it separate

Reply:
Ok!